src/main.rs

```diff
@@ -125,6 +131,7 @@ fn main() {
         ext_superscript: exts.remove("superscript"),
         ext_header_ids: matches.value_of("header-ids").map(|s| s.to_string()),
         ext_footnotes: matches.is_present("footnotes"),
+        ext_description_lists: matches.is_present("description-lists"),
```

Comment:
… and use the flag toggle here, not the `exts` set. As a result:

```console
$ cargo run <(echo "a"; echo; echo ": b") --description-lists
    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running `target/debug/comrak /dev/fd/63 --description-lists`
<dl><dt>
<p>a</p>
</dt>
<dd>
<p>b</p>
</dd>
</dl>
$ cargo run <(echo "a"; echo; echo ": b") -e description-lists
    Finished dev [unoptimized + debuginfo] target(s) in 0.06s
     Running `target/debug/comrak /dev/fd/63 -e description-lists`
thread 'main' panicked at 'assertion failed: exts.is_empty()', src/main.rs:137:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
$
```

Ideally description lists should be an `-e` option. (And so should footnotes, frankly.)

Reply:
Oops.. I didn't notice that =/.

Fixed in d7469e70d790ca5619faf5d4fe6f77a7a17a4c24.